azure/templates/pds-tests.yml

```diff
@@ -48,3 +48,24 @@ steps:
         $(Pipeline.Workspace)/s/$(SERVICE_NAME)/$(SERVICE_ARTIFACT_NAME)/tests/test-report.xml
         $(Pipeline.Workspace)/s/$(SERVICE_NAME)/$(SERVICE_ARTIFACT_NAME)/tests/*.xml
       failTaskOnFailedTests: true
+
+  - bash: poetry install
```

Comment:
I don't think this step is required, all the libs should have been installed already in an earlier step